### PR TITLE
Tech debt definition

### PR DESCRIPTION
**Pull request by andyblundell**

### CODE REVIEW

File: tech-debt.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+* This includes technical decisions that are made for short-term benefit (e.g. delivery time or cost) that will slow you down or increase your costs / risks over time
+* This also includes technical decisions that were appropriate at the time, but better options exist now (e.g. new technologies that weren't available at the time)
+
+Note: it includes working practices as well as the software: the scope includes anything that's not in line with the contents of this [quality framework](README.md)
+
+### Examples of what it IS
+
+Here is a non-exhaustive list of examples we consider to be *in* scope of "technical debt":
+
+* Unknown / poor code hygiene
```
Comment (stefaniuk) [inline tech-debt.md:36]: We can attribute the tech debt to the lack of coding standards. Can we change that bullet point to `Unknown / poor code hygiene and coding standards`, please?

Comment (andyblundell) [reply tech-debt.md]: Nice, thanks - done

**Review by stefaniuk — APPROVED** [review]
LGTM

File: tech-debt.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+### Examples of what it IS
+
+Here is a non-exhaustive list of examples we consider to be *in* scope of "technical debt":
+
+* Unknown / poor code hygiene and lack of coding standards
+* Unknown / poor [resilience test-automation](practices/service-reliability.md)
+* Unknown / poor [security test-automation](practices/security.md)
+* Unknown / poor accessibility test-automation
+* Use of technologies not endorsed by the organisation
```
Comment (walteck) [inline tech-debt.md:40]: Question: do we want to link to the tech radar here?

**Review by walteck — COMMENTED** [review]
General question as to whether we want to link out to the Tech Radar in this section to indicate where teams can view technology that is agreed for use in the organisation.

File: tech-debt.md (hunk 2 above)
Comment (andyblundell) [reply tech-debt.md:40]: Good question - we've just chatted and agreed: not at this time

**Pull request merged**